docs/static/breaking-changes.asciidoc

```diff
@@ -45,9 +47,30 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
  The new mapping template has been updated to reflect that. If you are using a custom mapping template you may need to update it to reflect that.
 
 [float]
+==== Kafka input changes
+
+* Updated to 0.10.1.0 client protocol. Only supports brokers 0.10.1.x or later
```

Comment:
This looks correct, but gives me pause to ask a question -- Kafka 0.11.x is the latest. Should we ship LS 6.0 with Kafka 0.11.x  by default?

(If Kafka 0.11.x server will work with a Kafka 0.10.x client, then we don't need to upgrade maybe)

Reply:
Actually, I think you are right here - we do ship with 0.11.x, which is backwards compatible with 0.10.1.x. I missed it, as it was in a non-breaking change